packages/storage/src/service.ts

```diff
@@ -63,8 +63,8 @@ export class Service {
    * bucket.
    */
   ref(path?: string): Reference {
-    function validator(path: string): void {
-      if (/^[A-Za-z]+:\/\//.test(path)) {
+    function validator(path: unknown): void {
+      if (/^[A-Za-z]+:\/\//.test(path as string)) {
```

Comment:
If you want to go the extra mile, then I would suggest we also validate explicitly that `path` is a string. Optional,

Reply:
Will give it a shot.

Reply:
Can you do the same thing for `refFromURL` below? Thanks!

Reply:
Done.